### ISSUE TITLE
Problems with IE6 in version 0.4

### ISSUE THREAD

**Issue by GoogleCodeExporter** [davidbelow/blueprintcss#10]
```
Try this with different browsers: http://peruncs.com/s4g/template_results.html

Firefox and Opera render the page OK, but IE6 - does not.

```

Original issue reported on code.google.com by `hr.stoya...@gmail.com` on 11 Aug 2007 at 7:41

**Comment by GoogleCodeExporter** [comment]
```
Thanks, I'll take a look.
```

Original comment by `ola...@gmail.com` on 13 Aug 2007 at 2:39

**Comment by GoogleCodeExporter** [comment]
```
This page does not render correctly in IE6 either:

   http://bjorkoy.com/blueprint/test.html
```

Original comment by `blodulv` on 21 Aug 2007 at 5:29

**Comment by GoogleCodeExporter** [comment]
```
html hr { margin: -0.7em 0 0.8em 0; } /* IE6 fix */

This was suggested by someone on a blog as a fix for the IE issues, which seem 
to
stem mostly from the use of the <hr /> element.

I think the use of 0.1em as a synonym for '1 pixel' is dangerous, however, and
sometimes does not work in Firefox when used there. I believe 0.083em might be
better, though I plan to test this.
```

Original comment by `blodulv` on 21 Aug 2007 at 10:10

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in 0.5.
```

Original comment by `ola...@gmail.com` on 22 Aug 2007 at 12:07
- Changed state: **Fixed**
